README.md

```diff
@@ -169,8 +169,17 @@ To test-drive your new Hyrax application in development mode, spin up the server
 ```
 rake hydra:server
 ```
+## Create default administrative set
+
+**After** Fedora and Solr are running, create the default administrative set -- into which all works will be deposited unless assigned to other administrative sets -- by running the following rake task:
+
+```
+rake hyrax:default_admin_set:create
+```
+
+**NOTE**: You will want to run this command the first time this code is deployed to a new environment as well.
 
-And now you should be able to browse to [localhost:3000](http://localhost:3000/) and see the application. Note that this web server is purely for development purposes; you will want to use a more fully featured [web server](#web-server) for production-like environments.
+After starting your servers and creating a default aministrative set you should be able to browse to [localhost:3000](http://localhost:3000/) and see the application. Note that this web server is purely for development purposes; you will want to use a more fully featured [web server](#web-server) for production-like environments.
```

Comment:
Hmmm... I ran the readme as written and waited 3-5 minutes for something to happen and nothing did.  I stopped the dev servers, restarted them, rand the default admin set task, and everything loaded pretty quickly.

I'll wipe my environment and try again with the original order of instructions and report back.  It't totally possible something else was going on that just looked like it got fixed by running the administrative set task.